package.vscode.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@apollo/client-devtools-vscode",
+  "version": "4.18.4-pre.0",
```

Comment:
That was just for my local build back when I forked off - when we publish the package, it will go through changesets and inherit the version of the extension :)